topics/full-stack-development-with-kotlin-multiplatform.topic

```diff
@@ -59,162 +59,111 @@
-                     alt="Project layout in Fleet" width="706" border-effect="rounded"/>
-                <p>
-                    It isn't immediately clear by looking at the project structure, but the
-                    <path>server</path>
-                    folder is one of three Kotlin modules in the project. The other two are
-                    <path>shared</path>
-                    and
-                    <path>composeApp</path>
-                    .
-                </p>
-                <p>
-                    The structure of the
-                    <path>server</path>
-                    module is very similar to that produced by the <a href="https://start.ktor.io/">Ktor Project
-                    Generator</a>.
-                    You have a dedicated build file to
-                    declare plugins and dependencies, and a source set containing the code to build and launch a Ktor
-                    service:
-                </p>
-                <img src="full_stack_development_tutorial_server_folder.png"
-                     alt="Contents of the server folder in a Kotlin Multiplatform project" width="300"
-                     border-effect="line"/>
-            </step>
-        </procedure>
-        <procedure title="Run the service" id="run-service">
-            <p>
-                You can run this service by executing the
-                <a href="https://www.jetbrains.com/help/fleet/run-configs.html">run configuration</a>
-                for the target.
-            </p>
-            <step>
-                Press
-                <shortcut>⌘Cmd+R</shortcut>
-                or select
-                <ui-path> Run | Run &amp; Debug</ui-path>
-                from the main menu to access the run configurations. A new popup window will open
-                displaying a list of available configurations.
-                <img src="full_stack_development_tutorial_run_configurations.png"
-                     alt="Run &amp; Debug window in Fleet" width="706"
-                     border-effect="rounded" style="block"/>
-
             </step>
             <step>
-                Click
-                <control>Run</control>
-                for the
-                <ui-path>server</ui-path>
-                configuration. The service will start in
-                a new
-                <ui-path>Terminal</ui-path>
-                panel at the bottom of the IDE.
+                Click on the
+                <ui-path>Run</ui-path>
+                button
+                (<img src="intellij_idea_gutter_icon.svg"
+                      style="inline" height="16" width="16"
+                      alt="IntelliJ IDEA run icon"/>)
+                next to the <code>main()</code> function to start the application.
+                <p>
+                    The service will start in a new
+                    <ui-path>Terminal</ui-path>
+                    panel at the bottom of the IDE.
```

Comment:
This should be either "Terminal tool window" or "new Terminal tab", as per IDEA control naming.